src/colvarbias_meta.cpp

```diff
@@ -331,13 +330,12 @@ colvarbias_meta::~colvarbias_meta()
   colvarbias_meta::clear_state_data();
   colvarproxy *proxy = cvm::proxy;
 
-  if (proxy->get_output_stream(replica_hills_file)) {
+  if (proxy->output_stream_exists(replica_hills_file)) {
     proxy->close_output_stream(replica_hills_file);
   }
```

Comment:
This could be done without testing for the existence first, since close_output_stream handles it gracefully.

Reply:
Done